CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Fixed
```

Comment:
Quite problematic to discover so many bugs after each release :sweat_smile: it means our test suite has caveats.
Can you try adding tests that would have caught these bugs?

Reply:
Yep I did not anticipate all the available cases. 😕